docs/wiki/archive/GPS-Rescue-v4-5.md

```diff
@@ -0,0 +1,526 @@
+# GPS Rescue 4.5
+
+Betaflight 4.5 makes the following significant improvements to GPS rescue:
+- greatly improved support for UBlox GPS modules, particularly M9 and M10, improving rescue reliability and smoothness.  
```

Comment:
Is this line worth capitalizing to make it adhere to the rest of the lines below? If so This line (4) as well as line 14 should be modified.

Reply:
good point, thank you